templates/post.html

```diff
@@ -1,20 +1,24 @@
 <div class="container">
-    <h2 class="text-center text-uppercase text-purple">$title$</h2>
-    <div class="row tag-block text-center">
-        <div class="col-md-12 col-xs-12 tag-block">
-            $for(tagsList)$
-            <span> <a class="btn btn-outline-dark" href="../tags/$tag$">$tag$</a></span>
-            $endfor$
+    <div class="row">
+        <div class="col-lg-8 col-md-10 mx-auto">
+            <h2 class="title">$title$</h2>
+            <div class="post-context">
+              <span class="author">Author Here | </span>
```

Comment:
If I remember, we decided to not mention the author name, so this part can be removed